pages/common/minetest.md

```diff
@@ -8,9 +8,9 @@
 
 `minetest`
 
-- Start minetest in server mode:
+- Start minetest in server mode by hosting a specific world:
 
-`minetest --server`
+`minetest --server --world {{name}}`
```

Comment:
This is a path to a world, not the name of the world. Specifically from the man page:

```
       --world <value>
              Set world path
```

Reply:
needs a new PR? cc @Yutyo